external/mmsegmentation/segmentation_tasks/extension/datasets/mmdataset.py

```diff
@@ -95,15 +95,18 @@ def __getitem__(self, index):
             :return data_info: dictionary that contains the image and image metadata, as well as the labels of
             the objects in the image
             """
-
             dataset = self.ote_dataset
             item = dataset[index]
 
+            label_idx = {label.id: i for i, label in enumerate(self.labels)}
```

Comment:
No need to make this id->idx table for each item; please refer to detection PR.

Reply:
Moved to init().